aidants_connect_web/admin.py

```diff
@@ -22,38 +23,71 @@
 admin_site = OTPAdminSite(OTPAdminSite.name)
 
 
-class AidantAdmin(DjangoUserAdmin):
-    # The forms to add and change aidant instances
+class VisibleToStaff(ModelAdmin):
+    """A mixin to make a ModelAdmin visible to staff users."""
```

Comment:
Si j'ai bien compris : 

```suggestion
    """A mixin to make a model registered in the Admin visible to staff users."""
```